resources/core/charts/k8s-dashboard-proxy/README.md

```diff
@@ -0,0 +1,23 @@
+# Kubernetes Dashboard Proxy
+
+## Overview
+
+The `Kubernetes Dashboard Proxy` is a transparent proxy for the `Kubernetes Dashboard`
+
+Read the [Kubernetes Dashboard Proxy](https://github.com/kyma-project/kyma/blob/master/components/k8s-dashboard-proxy/README.md) document to learn more about Kubernetes Dashboard Proxy in Kyma.
+
+This chart installs [Keycloak Proxy](https://github.com/keycloak/keycloak-gatekeeper) and `reverseproxy`, a thin layer Go application as a reverse proxy.
+
+Configure these options for each business requirement:
+
+| Component                 | Configuration  | Description |
+|---------------------------| --------------:| ----------: |
+| **k8s-dashboard-proxy** |
+| |`containerPort` | The proxy listen port. |
```

Comment:
```suggestion
| |`containerPort` | The port used by the proxy to listen on. |

```
I think this is a bit better - we can also go with `The proxy listening port` but I think the one above is a bit better.